kmultiversx/src/kmultiversx/kdist/mx-semantics/vmhooks/cryptoei.md

```diff
@@ -48,6 +48,23 @@ module CRYPTOEI-HELPERS
                 ...
         </instrs>
 
+    syntax InternalInstr ::= "#verifyEd25519"   [klabel(verifyEd25519), symbol]
+ // ---------------------------------------------------------------------------
+
+    rule [verifyEd25519]:
```

Comment:
What is this supposed to do? Right now, it's a verification that always succeeds.

Reply:
This was for a test I've decided not to include in this PR.